packages/chain/node.go

```diff
@@ -222,11 +236,14 @@ func New(
 		consRecoverPipe:        pipe.NewDefaultInfinitePipe(),
 		publishingTXes:         map[iotago.TransactionID]context.CancelFunc{},
 		procCache:              processors.MustNew(processorConfig),
+		configUpdatedCh:        make(chan *configUpdate, 1),
 		listener:               listener,
 		accessLock:             &sync.RWMutex{},
 		activeCommitteeDKShare: nil,
 		activeCommitteeNodes:   []*cryptolib.PublicKey{},
-		activeAccessNodes:      []*cryptolib.PublicKey{},
+		activeAccessNodes:      nil,                // updated bellow.
+		accessNodesFromNode:    accessNodesForNode, // TODO: Update it on event.
```

Comment:
```suggestion
		accessNodesFromNode:    accessNodesForNode,
```
we can probably remove the TODO

Reply:
Thanks!